boot.go

```diff
@@ -34,7 +34,7 @@ func init() {
 }
 
 func main() {
-	if os.Getenv("DEBUG") == "true" {
+	if os.Getenv("DEIS_DEBUG") == "true" {
```

Comment:
can this use `conf.Debug` (would have to be moved down to after the config is parsed)